tari-project/universe
# fix: when using tcp, use tcp seeds

Description
---
When not using tor, use the ip4, and ip6 DNS seed addresses.
Additionally, when the method of connection is changed clear out the peer databases to ensure no wasted efforts on connection types we don't support.

Fixes: #819 

Motivation and Context
---
Currently, our dns entry for seed nodes is overpopulated and can't be read by some ISP's due to the size of the txt records and timeouts.

This makes it nearly impossible to put seeds in for tor, ip4, and ip6 with enough for all. We have now separated out the addresses.
`seeds.esmeralda.tari.com` - with a mixture
`tor.seeds.esmeralda.tari.com` - with only tor addresses
`ip4.seeds.esmeralda.tari.com` - with only ip4 addresses
`ip6.seeds.esmeralda.tari.com` - with only ip6 addresses

When not connecting through tor, use the correct DNS address for seeds.

How Has This Been Tested?
---
Manually

Testing Instructions
---
- Start the application
- Turn off TOR
- See that you can still connect to the Tari network on restart
